flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -1255,3 +1257,30 @@ def set_if_not_none(dictionary, key, value):
 
     assert all(ems_constraints["derivative min"] == expected_site_production_capacity)
     assert all(ems_constraints["derivative max"] == expected_site_consumption_capacity)
+
+
+def test_build_device_soc_values(caplog):
+    caplog.set_level(logging.WARNING)
+    soc_values = [
+        {"datetime": datetime(2023, 5, 20, tzinfo=pytz.utc), "value": 1.0},
```

Comment:
Can you add one more target, but within the planning horizon, to check that it doesn't end up in the warning?